assets/locale/zh_CH.po

```diff
@@ -498,13 +498,13 @@ msgid "TDP Boost"
 msgstr "TDP超频"
 
 msgid "Manual Freq"
-msgstr "手动设置频率"
+msgstr "手动频率"
```

Comment:
If this is used to "describe the frequency value currently set manually by the user", I think it is OK.